linux/net/rina/ipcps/shim-tcp-udp.c

```diff
@@ -1032,6 +1032,22 @@ int send_msg(struct socket *      sock,
         return size;
 }
 
+char * tcp_udp_int_to_string(int data)
+{
+	char * result;
+
+        result = rkmalloc(10, GFP_ATOMIC);
```

Comment:
10 characters is usually plenty, but an int may need 12 (one for the - sign, 10 for the number and one for the \0)